src/main/java/org/fundacionjala/at15/katas/bankocr/vladimir/AccesoDatosEx.java

```diff
@@ -0,0 +1,7 @@
+package org.fundacionjala.at15.katas.bankocr.vladimir;
+
+public class AccesoDatosEx extends Exception {
```

Comment:
Please use english for all names and comments

Reply:
ok Jose, i will do it, thank you